diff --git a/kwant/linalg/mumps.py b/kwant/linalg/mumps.py
index c3bec6f..bc3a223 100644
--- a/kwant/linalg/mumps.py
+++ b/kwant/linalg/mumps.py
@@ -15,9 +15,19 @@ import time
 import numpy as np
 import scipy.sparse
 import warnings
-from . import _mumps
 from .fortran_helpers import prepare_for_fortran
 
+# This build of Kwant uses an MPI-enabled MUMPS. If we not ensure
+# that MPI is initialized here then MUMPS will crash the process.
+from mpi4py import MPI
+if MPI.Is_finalized() or not MPI.Is_initialized():
+    raise RuntimeError("MPI is not initialized.")
+del MPI  # Don't pollute the namespace.
+
+# This loads the Mumps shared library, so we must ensure that it happens
+# _after_ we ensure MPI is initialized, above.
+from . import _mumps
+
 orderings = { 'amd' : 0, 'amf' : 2, 'scotch' : 3, 'pord' : 4, 'metis' : 5,
               'qamd' : 6, 'auto' : 7 }
 
