diff --git a/kwant/tests/test_kpm.py b/kwant/tests/test_kpm.py
index 524cc57..5da8602 100644
--- a/kwant/tests/test_kpm.py
+++ b/kwant/tests/test_kpm.py
@@ -7,6 +7,7 @@
 # http://kwant-project.org/authors.

 from copy import copy as copy
+import sys
 from types import SimpleNamespace

 import pytest
@@ -404,6 +405,7 @@ def test_kwant_syst():
     assert_allclose(spectrum_syst.densities, spectrum_dense.densities)


+@pytest.mark.skipif(sys.platform == 'win32', reason="fails on conda-forge")
 def test_kwant_op():
     """Check that the kwant.operator.Density gives the same result as the
     identity operator.
